.github/workflows/connect-nextra-dev-release.yml

```diff
@@ -108,3 +109,64 @@ jobs:
           name: connect-explorer-nextra-webextension
           path: |
             packages/connect-explorer-nextra/build-webextension
+
+  methods:
+    needs: [build-deploy]
+    uses: ./.github/workflows/template-connect-popup-nextra-test-params.yml
+    with:
+      test-name: methods.test
+      DEV_SERVER_HOSTNAME: dev.suite.sldev.cz
+      run-webextension: true
+      build-overview: true
+
+  popup-close:
+    needs: [build-deploy]
+    uses: ./.github/workflows/template-connect-popup-nextra-test-params.yml
+    with:
+      test-name: popup-close.test
+      DEV_SERVER_HOSTNAME: dev.suite.sldev.cz
+      run-webextension: true
+
+  passphrase:
+    needs: [build-deploy]
+    uses: ./.github/workflows/template-connect-popup-nextra-test-params.yml
+    with:
+      test-name: passphrase.test
+      DEV_SERVER_HOSTNAME: dev.suite.sldev.cz
+      # currently flaky, run only in nightly
+      run-webextension: ${{ github.event_name == 'schedule' }}
```

Comment:
total nitpick: I'd maybe remove all webextension mentions here if it is not supported in the template..

Reply:
I kept them since I presume it will be eventually added to the template